### PR TITLE
Bootstrap Main Chapter Page

### PR DESCRIPTION
**Pull request by KimberleyCook**
Chapter Main page

| With Image and custom description | Default |
|-------|------|
|screencapture-localhost-3000-london-2020-11-23-19_26_17|screencapture-localhost-3000-london-2020-11-23-18_49_23|

### CODE REVIEW

File: app/assets/stylesheets/_bootstrap-custom.scss [hunk 1]
```diff
@@ -31,7 +31,7 @@
 // @import "bootstrap/breadcrumb";
 // @import "bootstrap/pagination";
 @import "bootstrap/badge";
-// @import "bootstrap/alert";
+@import "bootstrap/alert";
```
Comment (despo) [inline app/assets/stylesheets/_bootstrap-custom.scss:34]: how are we using alerts?

File: app/views/chapter/show.html.haml [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
         %p.lead= @chapter.description
       - else
         %p.lead= t('chapters.intro_html')
     - if @chapter.image.present?
-      .medium-6.columns.chapter-header-image
+      .col-12.col-md-6
         = image_tag @chapter.image.bg, alt: "#{@chapter.name} chapter"
 
-.panel
+.alert.alert-primary
```
Comment (KimberleyCook) [inline app/views/chapter/show.html.haml:13]: @despo I use alert here. It's not technically an alert but it looks like that component. Basically a div with coloured background and border.

Comment (despo) [reply app/views/chapter/show.html.haml:13]: 👍

**Comment by gnclmorais** [comment]
Possible idea for the future: automating our screenshots, so we can automatically see if something changed. I’ve used [Percy](https://percy.io/pricing) in the past and it was quite useful.

**Comment by KimberleyCook** [comment]
@gnclmorais oh nice, will look into this. Thanks!

**Comment by matyikriszta** [comment]
@KimberleyCook this is once again quite out of date 😭 Could you rebase it again?

**Pull request closed without merging**